src/test/java/api/requests/RequestsAPIUpdatingTests.java

```diff
@@ -544,6 +547,68 @@ public void requestCancellationNoticeIsSentWhenPolicyDefinesCancelledRequestNoti
     noticeContextMatchers.putAll(NoticeMatchers.getItemContextMatchers(temeraire, true));
     noticeContextMatchers.putAll(NoticeMatchers.getRequestContextMatchers(updatedRequest));
     noticeContextMatchers.putAll(NoticeMatchers.getCancelledRequestContextMatchers(updatedRequest));
+    noticeContextMatchers.put("request.reasonForCancellation",
+      is(itemNotAvailable.getJson().getString(CANCELLATION_REASON_PUBLIC_DESCRIPTION)));
+    MatcherAssert.assertThat(sentNotices,
```

Comment:
I find it more readable when `assertThat` is statically imported

Reply:
replaced by static imports